src/test/java/com/arangodb/ArangoDBTest.java

```diff
@@ -83,105 +83,138 @@ public static void shutdown() {
+    }
+
+    @Test
 	public void createDatabaseWithOptionsSatellite() {
-		assumeTrue(isCluster());
-		assumeTrue(isEnterprise());
-		assumeTrue(isAtLeastVersion(3, 6));
-
-		final String dbName = "testDB-" + UUID.randomUUID().toString();
-		final Boolean resultCreate = arangoDB.createDatabase(new DBCreateOptions()
-				.name(dbName)
-				.options(new DatabaseOptions()
-						.writeConcern(2)
-						.satellite(true)
-						.sharding("")
-				)
-		);
-		assertThat(resultCreate, is(true));
-
-		DatabaseEntity info = arangoDB.db(dbName).getInfo();
-		assertThat(info.getReplicationFactor(), nullValue());
-		assertThat(info.getSatellite(), is(true));
-		assertThat(info.getWriteConcern(), is(2));
-		assertThat(info.getSharding(), is(""));
-
-		final Boolean resultDelete = arangoDB.db(dbName).drop();
-		assertThat(resultDelete, is(true));
-	}
-
-	@Test
-	public void getDatabases() {
-		Collection<String> dbs = arangoDB.getDatabases();
-		assertThat(dbs, is(notNullValue()));
-		assertThat(dbs.size(), is(greaterThan(0)));
-		assertThat(dbs.contains("_system"), is(true));
-		assertThat(dbs, hasItem(DB1));
-	}
+        assumeTrue(isCluster());
+        assumeTrue(isEnterprise());
+        assumeTrue(isAtLeastVersion(3, 6));
+
+        final String dbName = "testDB-" + UUID.randomUUID().toString();
+        final Boolean resultCreate = arangoDB.createDatabase(new DBCreateOptions()
+                .name(dbName)
+                .options(new DatabaseOptions()
+                        .writeConcern(2)
+                        .satellite(true)
+                        .sharding("")
+                )
+        );
+        assertThat(resultCreate, is(true));
+
+        DatabaseEntity info = arangoDB.db(dbName).getInfo();
+        assertThat(info.getReplicationFactor(), nullValue());
+        assertThat(info.getSatellite(), is(true));
+        assertThat(info.getWriteConcern(), is(2));
+        assertThat(info.getSharding(), is(""));
+
+        final Boolean resultDelete = arangoDB.db(dbName).drop();
+        assertThat(resultDelete, is(true));
+    }
+
+    @Test
+    public void createDatabaseWithUsers() {
+        final String dbName = "testDB-" + UUID.randomUUID().toString();
+
+        final Boolean resultCreate = arangoDB.createDatabase(new DBCreateOptions()
+                .name(dbName)
+                .users(Collections.singletonList(new DatabaseUsersOptions()
+                        .active(true)
+                        .username("testUser")
+                        .passwd("testPasswd")
+                ))
+        );
+        assertThat(resultCreate, is(true));
+
+        DatabaseEntity info = arangoDB.db(dbName).getInfo();
+        assertThat(info.getName(), is(dbName));
+
+        assertThat(arangoDB.getUsers().stream().anyMatch(it -> it.getUser().equals("testUser")), is(true));
+
+        ArangoDB arangoDBTestUser = new ArangoDB.Builder()
+                .user("testUser")
+                .password("testPasswd")
+                .build();
```

Comment:
Should you include extra in the test as well?

Reply:
Thanks for spotting it, I added an assertion to check it.